### PR TITLE
Apply redesign and re-order departments

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Applied styling updates based on [mock-up](https://www.figma.com/file/UsovUAf4zELQgGd9bxhiVe/Careers?node-id=310%3A1822&t=JxZNmeLpAdpskinS-0)
- Applied requested order to departments

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://canonical-com-697.demos.haus/careers/all
- See that the page matches the mock-up and that filters work as expected

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-677

### CODE REVIEW

**Comment by mtruj013** [comment]
Blocked by missing departments. The requested order is as follows:  
Screenshot from 2022-11-24 18-39-37

However the API does not return support-engineering, legal, or administration as departments and it does return techops (which is not on the mock-up). I've replaced support-engineering with techops for now.

cc: @marinacastejon @anthonydillon

**Comment by anthonydillon** [comment]
That is correct, tech ops becomes support engineering.

File: templates/careers/all.html [hunk 1]
```diff
@@ -7,16 +7,14 @@
 {% block meta_description %}Canonical publishes Ubuntu, provides commercial services and solutions for Ubuntu, and works with hardware manufacturers, software vendors and public.{% endblock %}
 
 {% block hero %}
-  <section class="p-strip is-deep">
-    <div class="row">
-      <div class="col-3">
+  <section class="p-strip--light is-deep">
```
Comment (lyubomir-popov) [inline templates/careers/all.html:10]:
```suggestion
  <section class="p-strip--light">
```

File: templates/careers/base.html [hunk 2]
```diff
@@ -8,8 +8,6 @@
 {% block hero %}{% endblock %}
 
 <section class="p-strip u-no-padding--top u-extra-space">
```
Comment (lyubomir-popov) [inline templates/careers/base.html:10]:
```suggestion
<section class="p-strip u-extra-space">
```

**Review by lyubomir-popov — APPROVED** [review]
Looks good just 2 suggestions to imprvoe the white space

**Review by anthonydillon — APPROVED** [review]
Thanks @mtruj013 LGTM :+1:

**Pull request merged**